docs/source/glossaries.rst

```diff
@@ -0,0 +1,107 @@
+Glossaries
+**********
+
+=========== ==============================================================
+Keyword     Description                                                   
+=========== ==============================================================
+ACK         Acknowledge                                                   
+am          Active Message                                                
+AMO         Atomic Memory Operation                                       
+ANL         Argonne National Laboratory                                   
+AZP         AZure Pipeline                                                
+bcopy       Byte copy                                                     
+Bistro      Binary Instrumentation                                        
+BTL         Byte Transfer Layer                                           
+cm          Connection Manager                                            
+CMA         Cross Memory Attach                                           
+CQ          Completion Queue(Infiniband)                                  
+CQE         Completion Queue Entry(Infiniband)                            
+csmock      static analysis tools                                         
+CUDA        Compute Unified Device Architecture(NVIDIA)                   
+DC          Dynamically Connected transport(Infiniband)                   
+DPM         Direct Packet Mode                                            
+ep          EndPoint                                                      
+FC          Flow Control                                                  
+fd          File Descriptor                                               
+GDR         GPUDirect RDMA                                                
+gtest       Google Test                                                   
+HPC         High Performance Computing                                    
+HWTM        HardWare Tag Matching                                         
+IB          Infiniband                                                    
+iface       Interaface                                                    
+IPC         Inter Process Communication                                   
+JUCX        Java API over UCP                                             
+KLM         A new sophisticated way of creating memory regions.(Mellanox) 
+KNEM        Kernel Nemesis                                                
+LDPM        Legacy DPM                                                    
+LLNL        Lawrence Livermore National Laboratory                        
+madvise     give advice about use of memory. See manual madvise(2)        
+md          Memory Domain                                                 
+MEMH        Memory Handle                                                 
+MLX         Mellanox Technologies                                         
+mlx5        Connect-X5 VPI                                                
```

Comment:
Mellanox-5 generation of Mellanox device driver (I think all connectX-4 generation and above)